specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -1930,6 +2076,100 @@
         }
       },
       "description": "The backup key result, containing the backup blob."
+    },
+    "KeyPolicy": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The key policy id.",
+          "readOnly": true
+        },
+        "lifetimeActions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LifetimeActions"
+          },
+          "description": "Actions that will be performed by Key Vault over the lifetime of a key."
+        },
+        "attributes": {
+          "$ref": "#/definitions/KeyPolicyAttributes",
+          "description": "The key policy attributes."
+        }
+      },
+      "description": "Management policy for a key."
+    },
+    "LifetimeActions": {
+      "properties": {
+        "trigger": {
+          "$ref": "#/definitions/LifetimeActionsTrigger",
+          "description": "The condition that will execute the action."
+        },
+        "action": {
+          "$ref": "#/definitions/LifetimeActionsType",
+          "description": "The action that will be executed."
+        }
+      },
+      "description": "Action and its trigger that will be performed by Key Vault over the lifetime of a key."
+    },
+    "LifetimeActionsTrigger": {
+      "properties": {
+        "timeAfterCreation": {
+          "type": "string",
+          "format": "string",
+          "minimum": 1,
+          "maximum": 99,
+          "description": "Days after creation to attempt rotate."
+        },
+        "timeBeforeExpiry": {
+          "type": "string",
+          "format": "string",
+          "description": "Days before expiry to attempt rotate."
+        }
+      },
+      "description": "A condition to be satisfied for an action to be executed."
+    },
+    "LifetimeActionsType": {
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of the action.",
+          "enum": [
+            "Rotate",
+            "Notify"
+          ],
+          "x-ms-enum": {
+            "name": "ActionType",
+            "modelAsString": false
+          }
+        }
+      },
+      "description": "The action that will be executed."
+    },
+    "KeyPolicyAttributes": {
+      "properties": {
+        "rotationEnabled": {
+          "type": "boolean",
+          "description": "Determines whether key auto rotation is enabled."
+        },
+        "rotationTime": {
+          "type": "string",
+          "format": "string",
+          "description": "Time to attempt rotate."
```

Comment:
You should note that this is in a particular format (e.g. "1Y", "90D", etc.).

Reply:
Format example has been added